jni/nnstreamer.mk

```diff
@@ -9,10 +9,16 @@ $(error GSTREAMER_ROOT_ANDROID is not defined!)
 endif
 
 NNSTREAMER_VERSION  := 1.5.2
+NNSTREAMER_VERSION_MAJOR := $(word 1,$(subst ., ,${NNSTREAMER_VERSION}))
+NNSTREAMER_VERSION_MINOR := $(word 2,$(subst ., ,${NNSTREAMER_VERSION}))
+NNSTREAMER_VERSION_MICRO := $(word 3,$(subst ., ,${NNSTREAMER_VERSION}))
 
 NNSTREAMER_GST_HOME := $(NNSTREAMER_ROOT)/gst/nnstreamer
 NNSTREAMER_EXT_HOME := $(NNSTREAMER_ROOT)/ext/nnstreamer
 NNSTREAMER_CAPI_HOME := $(NNSTREAMER_ROOT)/api/capi
+sed -i "s/@__NNSTREAMER_VERSION_MAJOR__@/${NNSTREAMER_VERSION_MAJOR}/" ${NNSTREAMER_GST_HOME}/include/nnstreamer_plugin_api.h
```

Comment:
this block should be in build-android-lib.sh

Reply:
That bash script won't be aware of the version numbers... :(
We need to execute sed in Android-Makefile somehow. (which this script is failing :( )

Reply:
Or.. maybe I'll do grep on nnstreamer.mk from build-android-lib.sh :)

Reply:
> this block should be in build-android-lib.sh

No. In CI, it build nnstreamer.mk without build-android-libs.sh and it's safer to assume that a user may build directly with .mk files. This will support both building with the shell script and building directly with .mk files